docs/transpile/rest-api-transpilation.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Transpilation using REST APIs
+description: How to transpile quantum circuits using REST APIs.
+---
+
+# Transpilation using REST APIs
+
+The circuit provided will need to be mapped to instructions fitting to the hardware basis instructions of the QPU, which is a process known as Transpilation.
+
+* You can either transpile [locally via Qiskit](https://docs.quantum.ibm.com/transpile/defaults-and-configuration-options) before generating the QASM string, or:
+
+* Use the [Cloud Transpiler API](https://cloud-transpiler.quantum.ibm.com/docs)
+
+<Admonition type="note">
+    [Transpilation is necessary](https://docs.quantum.ibm.com/announcements/product-updates/2024-02-14-qiskit-runtime-primitives-update) prior to submitting a circuit to IBM Quantum backends. 
+</Admonition>
+
+
+### Query the Cloud Transpilation service
+
+Query the [Cloud Transpilation service](https://cloud-transpiler.quantum-computing.ibm.com/docs) and provide your QASM string as input. See more details and documentation [here](https://docs.quantum.ibm.com/api/qiskit-transpiler-service-rest). 
+
+<Admonition type="note">
+    This experimental service is only available for IBM Quantum Premium Plan users.
+</Admonition>
+
+<Tabs>
+    <TabItem value="Qiskit Transpiler" label="Qiskit Transpiler">
+        ```python
+backend='ibm_brisbane' #choose your IBM Backend
+
+headers = {
+    "accept": "application/json",
+    "Authorization": f"Bearer {token}",
+    "Content-Type": "application/json",
+        }
+body= {
+    "qasm_circuits": qasm_string,
+}
+params = {
+        "backend": backend,
+        "optimization_level": 1,
+        "ai": "false", # "true", "false" and "auto"
+        }
+
+resp = requests.post(
+            "https://cloud-transpiler.quantum.ibm.com/transpile",
+            headers=headers,
+            json=body,
+            params=params,
+        )
+        ```
+    </TabItem>
+
+    <TabItem value="AI Transpiler" label="AI Transpiler">
```

Comment:
we're note supposed to use the term "AI transpiler" in official docs, instead we should say "AI enhanced transpiler" or "AI enahnced transpiler passes"